client/src/components/Workbench/Workbench.scss

```diff
@@ -47,6 +47,17 @@
 .workbench-content .wb-link {
   color: var(--global-color-accent--normal);
 }
+.workbench-content .wb-link:has(.icon):hover {
+  text-decoration: unset;
+}
+
+.workbench-content .wb-link:has(.icon) > span {
+  text-decoration: unset;
+}
```

Comment:
This line appears unnecessary. It's presence may be the fault of my copy-pasta or draft code suggestion.

<details><summary>Testing</summary>

1. I added the proposed code to the proposed file.
    - I did this via Developer Tools, not local dev env (which is where I should test).
3. I removed this line.
4. The bug is still fixed.

</details>

```suggestion
```

Reply:
I am resolving this, because code has changed. If the comment is still valid, I will re-add it.